Can't use `async` or `Promise` in the playground: it errors complaining about an older `lib` target

As of 2023-10-30, when entering this code in the playground:
```ts
/** @gqlType */
class Query {
  /**
   * @gqlField
   */
  async test(): Promise<string> {
    return '';
  }
}
```
We get an error about `Promise` not being defined:
```
# ERROR MESSAGE
# =============

# index.ts:6:17 - error TS2705: An async function or method in ES5/ES3 requires the 'Promise' constructor.  Make sure you have a declaration for the 'Promise' constructor or include 'ES2015' in your '--lib' option.
# 
# 6   async test(): Promise<string> {
#                   ~~~~~~~~~~~~~~~
# 
```
I thought this was going to be an easy thing to fix, except, it looks like the playground's TS config [already targets a recent environment](https://github.com/captbaritone/grats/blob/3e9d666002a5eef96cc2d5364dddb20d5af6535c/website/src/components/PlaygroundFeatures/editors/InputView.tsx#L50) (ES2021)?

```ts
    createDefaultMapFromCDN(
      { target: ts.ScriptTarget.ES2021, lib: ["es2021"] },
      ts.version,
      shouldCache,
      ts,
      lzstring,
    )
```

However, I also found this [commented line](https://github.com/captbaritone/grats/blob/3e9d666002a5eef96cc2d5364dddb20d5af6535c/website/src/components/PlaygroundFeatures/linter.ts#L37):
```
  const compilerOpts = {
    allowJs: true,
    baseUrl: "./",
    paths: { grats: [GRATS_PATH] },
    // lib: ["es2021"],
  };
  const host = createVirtualCompilerHost(system, compilerOpts, ts);
```
I'm curious what is the Chesterton's Fence reason for having `lib: ["es2021"]` in one component but not in the other?

--- a/TODO.md
+++ b/TODO.md
@@ -14,6 +14,7 @@
 - [ ] Write guides
   - [ ] DataLoader pattern
   - [ ] Subscriptions
+- [ ] Playground does not support promises
 - [ ] Collect deprecated tags on all nodes and let GraphQL validation report an error
 - [ ] Consider checking for @gql in non-docblock comments
 - Example that includes a mutation and query

--- a/website/package.json
+++ b/website/package.json
@@ -30,7 +30,7 @@
     "@graphql-tools/utils": "^9.2.1",
     "@mdx-js/react": "^1.6.22",
     "@types/node": "^18.15.0",
-    "@typescript/vfs": "1.5.0",
+    "@typescript/vfs": "^1.4.0",
     "clsx": "^1.2.1",
     "cm6-graphql": "^0.0.3",
     "glob": "^9.3.4",

--- a/website/src/components/PlaygroundFeatures/linter.ts
+++ b/website/src/components/PlaygroundFeatures/linter.ts
@@ -34,7 +34,7 @@ function buildSchemaResultWithFsMap(fsMap, text, config) {
     allowJs: true,
     baseUrl: "./",
     paths: { grats: [GRATS_PATH] },
-    lib: [...fsMap.keys()],
+    // lib: ["es2021"],
   };
   const host = createVirtualCompilerHost(system, compilerOpts, ts);
 

